test/util/etcd.go

```diff
@@ -52,7 +52,7 @@ func GetEtcdURL() string {
 		return etcdFromEnv
 	}
 
-	etcdPort := "4001"
+	etcdPort := "44001"
```

Comment:
as soon as this util class is used for multiple test scripts, picking a special different port won't help with parallelization